pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -97,6 +97,11 @@ stdenv.mkDerivation {
   outputs = [ "out" "man" ];
 
   passthru = {
+    # "cc" is the generic name for a C compiler, but there is no one for package
+    # providing the linker and related tools. The two we use now are GNU
+    # Binutils, and Apple's "cctools"; "binutils" as an attempt to find an
+    # unused middle-ground name that evokes both.
+    bintools = binutils_bin;
```

Comment:
@copumpkin I ended up going with "bintools" rather than "utils", for the reasons above, but the concept is the same as what I mentioned in https://github.com/NixOS/nixpkgs/pull/29396#issuecomment-333688023 .